letsdrill.rb

```diff
@@ -1,12 +1,30 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  case integer
+  when 0..59
+    p "F"
+  when 60..69
+    p "D"
+  when 70..79
+    p "C"
+  when 80..89
+    p "B"
+  when 90..100
+    p "A"
+  begin
```

Comment:
Is this intended to stay in the method or was this some debugging code?

Nice job using a case statement